bazel-mode.el

```diff
@@ -21,6 +21,31 @@
 ;;
 ;;; Code:
 
+(defcustom buildifier-cmd "buildifier"
```

Comment:
Please start all names with the name of the package, as Emacs Lisp doesn't have namespaces. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html.

Reply:
Done.

Reply:
Ah, that was foolish of me. I instead read this as a request to make the interactive command internal, which I found confusing and should've double checked that's what you meant. Fixed.